V6_MIGRATION.md

```diff
@@ -32,7 +32,11 @@ carthage update --use-xcframeworks
 ## Venmo
 `BTVenmoDriver` has been renamed to `BTVenmoClient`
 
-`BTVenmoRequest` must now be initialized with a `paymentMethodUsage`. 
+All properties within `BTVenmoRequest` are now internal and included in the initializer.
+
+```
+let venmoRequest =  BTVenmoRequest(paymentMethodUsage: .multiUse, vault: true,fallbackToWeb: true)
```

Comment:
I think we want this to go in `V7_MIGRATION` guide. We can start one up to start adding changes to and use the v6 one as a template for formatting.

Reply:
Yes. The V6_MIGRATION guide is for merchants going from V5 --> V6, and the V7_MIGRATION guide will be for merchants going from V6 --> V7

Reply:
ahhh yes that makes sense, thanks y'all - addressed in 1081b0cf9e74b4c2a3278fc7a14658181baf6947